src/main/java/pl/llp/aircasting/screens/common/ToggleAircastingManagerFactory.java

```diff
@@ -15,17 +15,12 @@
  * Created by radek on 13/07/17.
  */
 public class ToggleAircastingManagerFactory {
-    @Inject
-    CurrentSessionManager currentSessionManager;
-    @Inject
-    SettingsHelper settingsHelper;
-    @Inject
-    LocationHelper locationHelper;
-    @Inject
-    CurrentSessionSensorManager currentSessionSensorManager;
+    @Inject CurrentSessionManager currentSessionManager;
```

Comment:
Is `@Inject` doing some dependency inj dance?

Reply:
Yeah, it's Roboguice doing its magic. The project is retired, we are using an old version, but it actually works well enough, that I don't see a good reason to upgrade or replace it. But it basically provides all the objects you need for the initialization of another objects, provided that this object has a 0-arguments constructor. But for some reason I couldn't do that with `ToggleAircastingManager`, so I inject all those into a factory like so, and then create it.